gui.py

```diff
@@ -0,0 +1,246 @@
+        # Insert mock data
+        self.tree.insert("", "end", values=("johndoe", "9:00 AM", "5:00 PM", 8))
+        self.tree.insert("", "end", values=("janedoe", "8:00 AM", "4:00 PM", 8))
+
+        # Add buttons for Edit Employee and Reports
+        edit_button = tk.Button(self.admin_frame, text="Edit Employee", command=self.open_edit_employee_window,
+                                bg="#00796B", fg="white", font=("Roboto", 12, "bold"), relief="flat", bd=0)
+        edit_button.pack(anchor="center", pady=10)
+
+        report_button = tk.Button(self.admin_frame, text="Reports", command=self.open_reports_window,
+                                  bg="#00796B", fg="white", font=("Roboto", 12, "bold"), relief="flat", bd=0)
+        report_button.pack(anchor="center", pady=10)
+
+    def create_reports_screen(self):
+        # Title for reports
+        reports_title = ttk.Label(self.reports_frame, text="Reports",
+                                  font=("Roboto", 20, "bold"), foreground="#00796B")
+        reports_title.pack(pady=20)
+
+        # Placeholder for future report content
+        report_label = ttk.Label(self.reports_frame, text="Reports will be displayed here.", font=("Roboto", 12))
+        report_label.pack(pady=20)
+
+    def create_log_in_screen(self):
+        title_label = ttk.Label(self.log_in_frame, text=f"{self.username.get()} is logged in",
+                                font=("Roboto", 20, "bold"), foreground="#00796B")
+        title_label.pack(pady=20)
+        # Clock in/out button
+        self.clock_in_button = tk.Button(self.log_in_frame, text="Clock in", command=self.clock_in,
+                                         bg="#00796B", fg="white", font=("Roboto", 12, "bold"),
+                                         relief="flat", bd=0, height=2, width=20, cursor="hand2")
+        self.clock_in_button.pack(anchor="center", pady=20)
+
+
+
+        # Total hours worked
+        self.total_hours_label = ttk.Label(self.log_in_frame, text="Total hours: 0", font=("Roboto", 12))
+        self.total_hours_label.pack(anchor="center", pady=10)
+
+    def open_edit_employee_window(self):
+        """Opens a new window to edit employee details."""
+        edit_window = tk.Toplevel(self.root)
+        edit_window.title("Edit Employee")
+        edit_window.geometry("400x300")
+
+        # Example in-line editable employee table (simple example)
+        edit_label = ttk.Label(edit_window, text="Edit employee data here.")
+        edit_label.pack(pady=20)
+
+    def open_reports_window(self):
+        """Opens a new window to view and print reports."""
+        # if self.logged_in_user.is _admin()== False:
+            # return with an error message
+
+        report_window = tk.Toplevel(self.root)
+        report_window.title("Reports")
+        report_window.geometry("500x400")
+
+        # Range of dates and employee selection
+        range_label = ttk.Label(report_window, text="Specify Date Range", font=("Roboto", 12))
```

Comment:
The input for Specify Date Range for Reports on the Admin panel should be a date time type of input and not an open ended input